rudder-core/src/main/scala/com/normation/rudder/datasources/DataSourceRepository.scala

```diff
@@ -133,6 +133,16 @@ class DataSourceRepoImpl(
 
   private[this] var datasources = Map[DataSourceId, DataSourceScheduler]()
 
+  // Initialize data sources scheduler, with all sources present in backend
+  getAll match {
```

Comment:
And so, in Rudder the solution is to use a ```BootstrapChecks``` (that will give nice init log for free)